rfcs/0001-meta-schedule-autotensorir.md

```diff
@@ -0,0 +1,444 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+* Feature Name: Meta Schedule (Formerly AutoTIR)
+* Start Date: 2021-05-28
+* RFC PR: TBD (apache/tvm-rfcs#0000)
+* GitHub Issue: TBD (apache/tvm-rfcs#0000)
+
+## 1. Summary
+
+This proposal introduces Meta Schedule: a probabilistic scheduling DSL on TIR that unifies the
+approaches of AutoTVM and Auto Scheduler (Ansor). Meta schedule provides a pragmatic way to define
+the space of automatic tuning, extensibility in terms of all possible TIR schedule primitives like
+tensorization and loop partitioning, and customizability on every layer of the automation system.
+
+Meta Schedule is our 3rd generation automatic scheduling system.
+
+## 2. Motivation
+
+**Scheduling and design space.** In TVM TensorIR, optimization of a TensorIR program is done via a
+sequence of transformations. For example, we reorder loops for better locality and we tensorize for
+specific hardware intrinsics. The process of invoking such a set of pre-defined transformations is
+called "**scheduling**", and each transformation is called a "**schedule primitive**". These
+primitives form a domain-specific language (DSL) describing the transformation of TensorIR programs.
+**Design space** is the set of all possible schedulings with respect to a TensorIR program.
+
+**Problems with the current scheduling system.** Currently we have 3 sets of scheduling APIs:
```

Comment:
Yeah I was a bit uncertain if we need a subheading because it is a single paragraph when drafting this RFC, but it seems all of us prefer it to be subheading so will do. Thanks!